templates/overrides/paragaphs/paragraph--sa-faq.html.twig

```diff
@@ -1,8 +1,7 @@
 {% extends "paragraph.html.twig" %}
 {% block content %}
-	{{ content.sa_header }}
```

Comment:
Why are these being removed?

Reply:
Since it is extending, those fields are being loaded in the parent template as well so it was creating a duplicate title and description